cfn.properties

```diff
@@ -0,0 +1,14 @@
+name=Cloudformation Plugin for SonarQube
```

Comment:
You've named this file `cfn.properties` but the name needs to match your plugin key, which is `cloudformation` according to your jar manifest

Reply:
renamed property file to cloudformation.